documentation/staging/content/samples/domains/model-in-image/initial.md

```diff
@@ -490,249 +490,274 @@ Now, you create a Domain YAML file. A Domain is the key resource that tells the
-          value: "-XX:+UseContainerSupport -Djava.security.egd=file:/dev/./urandom "
-
-        # Optional volumes and mounts for the domain's pods. See also 'logHome'.
-        #volumes:
-        #- name: weblogic-domain-storage-volume
-        #  persistentVolumeClaim:
-        #    claimName: sample-domain1-weblogic-sample-pvc
-        #volumeMounts:
-        #- mountPath: /shared
-        #  name: weblogic-domain-storage-volume
-
-      # The desired behavior for starting the domain's administration server.
-      # adminServer:
-        # Setup a Kubernetes node port for the administration server default channel
-        #adminService:
-        #  channels:
-        #  - channelName: default
-        #    nodePort: 30701
-
-      # The number of managed servers to start for unlisted clusters
-      replicas: 1
-
-      # The desired behavior for starting a specific cluster's member servers
-      clusters:
-      - clusterName: cluster-1
-        replicas: 2
-
-      # Change the `restartVersion` to force the introspector job to rerun
-      # and apply any new model configuration, to also force a subsequent
-      # roll of your domain's WebLogic Server pods.
-      restartVersion: '1'
-
-      configuration:
-
-        # Settings for domainHomeSourceType 'FromModel'
-        model:
-          # Valid model domain types are 'WLS', 'JRF', and 'RestrictedJRF', default is 'WLS'
-          domainType: "WLS"
-
-          # Optional configmap for additional models and variable files
-          #configMap: sample-domain1-wdt-config-map
-
-          # All 'FromModel' domains require a runtimeEncryptionSecret with a 'password' field
-          runtimeEncryptionSecret: sample-domain1-runtime-encryption-secret
-
-        # Secrets that are referenced by model yaml macros
-        # (the model yaml in the optional configMap or in the image)
-        #secrets:
-        #- sample-domain1-datasource-secret
-  ```
-  {{% /expand %}}
+{{%expand "Click here to view the WLS Domain YAML file." %}}
+```yaml
+# Copyright (c) 2020, 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# This is an example of how to define a Domain resource.
+#
+apiVersion: "weblogic.oracle/v9"
+kind: Domain
+metadata:
+  name: sample-domain1
+  namespace: sample-domain1-ns
+  labels:
+    weblogic.domainUID: sample-domain1
+
+spec:
+  # Set to 'FromModel' to indicate 'Model in Image'.
+  domainHomeSourceType: FromModel
+
+  # The WebLogic Domain Home, this must be a location within
+  # the image for 'Model in Image' domains.
+  domainHome: /u01/domains/sample-domain1
+
+  # The WebLogic Server image that the Operator uses to start the domain
+  image: "model-in-image:WLS-v1"
+
+  # Defaults to "Always" if image tag (version) is ':latest'
+  imagePullPolicy: IfNotPresent
+
+  # Identify which Secret contains the credentials for pulling an image
+  #imagePullSecrets:
+  #- name: regsecret
```

Comment:
Should this have regsecret2 since it's in the other templates?

Reply:
No. This is not an 'aux image' case (which potentially needs a second secret)...